grammars/pegcoffee.cson

```diff
@@ -62,8 +62,26 @@
 'repository':
   'charSet':
     'begin': '\\['
-    'end': '\\]'
+    'end': '\\](i?)'
+    'beginCaptures':
+      '0':
+        'name': 'keyword.operator'
+    'endCaptures':
+      '0':
+        'name': 'keyword.operator'
+      '1':
+        'name': 'constant.character.control.regexp'
     'name': 'keyword.operator'
```

Comment:
This is no longer accurate (but could be replaced with something like `meta.character-class`).

Reply:
This still needs to be changed to something that better conveys the semantic meaning, right?

Reply:
now using meta.brace and keyword.operator to separate the two